Listenbrainz/Repository/Yim/YimRepositoryImpl.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  YimRepositoryImpl.swift
+//  Listenbrainz
+//
+//  Created by Gaurav Bhardwaj on 11/01/24.
+//
+
+import Combine
+import Alamofire
+
+class YIMRepositoryImpl: YIMRepository {
+    func fetchYIMData(userName:String) -> AnyPublisher<YIMData, Error> {
+        let url = "https://beta-api.listenbrainz.org/1/stats/user/\(userName)/year-in-music/2023"
```

Comment:
@gauravbhardwaj7 Please keep this in the constants file as BETA_LISTENBRAINZ_BASE_URL